### PR TITLE
Remove babel

### LINKED ISSUE #11: Remove babel
Just use what is available in node 4.x

### PR DESCRIPTION
**Pull request by franvarney**
Closes #11

### CODE REVIEW

File: server/handlers/project.js [hunk 1]
```diff
@@ -1,12 +1,12 @@
-import {badRequest, notFound} from 'boom'
-import Joi from 'joi'
-import Logger from '@modulus/logger'
+'use strict'
```
Comment (jackboberg) [inline server/handlers/project.js]: why `use strict`?

Comment (franvarney) [inline server/handlers/project.js]: So I can use `let`

File: server/routes.js [hunk 2]
```diff
@@ -12,7 +12,7 @@ export default [
   { method: 'GET', path: '/projects', config: Project.getAll },
   { method: 'PUT', path: '/projects/{name}', config: Project.update },
 
-  // Versions
+  // // Versions
```
Comment (jackboberg) [inline server/routes.js]: double comment?

**Comment by jackboberg** [comment]
:+1:

**Comment by tzmanics** [comment]
:+1:

**Pull request merged**